### DISCUSSION TITLE
Are all Pins needed for GreaseWeazle?

### DISCUSSION THREAD

**Discussion by ScytheandGenesisGames** [keirf/greaseweazle#209] [Q&A] [answered]
Most of the drives I'm looking for which a TEAC brand must have pin 3 removed - is this needed for the program to work? I'm basically gonna find some with all pins on the floppy drive.

**Comment by keirf** [comment, marked as answer]
Gotek pin 3 is not used. You can cut it off if you want.

**Reply by ScytheandGenesisGames** [reply]
okay just checking I'll just leave it on for my Drives I have when I get the board from tindie.
